scripts/qiita-private-launcher

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Looking back over the earlier elements of the commit and maybe making a bit more sense to me now. But what exactly is a private task in the context of the typical qiita execution?